Sudoku/Sudoku-Puzzle/org.gnome.Sudoku-Puzzle.json

```diff
@@ -0,0 +1,86 @@
+{
+  "app-id": "org.gnome.Sudoku-Puzzle",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "sudoku-puzzle",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk9"
+  ],
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
You're going to want to add:
>     "--env=PATH=/usr/bin:/app/bin:/usr/lib/sdk/openjdk9/bin"
here

Reply:
still I get "./build.sh: line 3: javac: command not found"
after adding also "./build.sh" in 
"name": "sudoku-puzzle",
      "buildsystem": "simple",
      "build-commands": [ "./build.sh",
	"rm -rf ./bin/META-INF"
      ],

Reply:
finish args are for run time not build time.

Reply:
I think that the pb is that javac is not included in the sandbox

Reply:
Maybe this fixes it: https://github.com/flathub/org.octave.Octave/blob/master/org.octave.Octave.json#L27

Reply:
Now I get ...

Running: /usr/lib/sdk/openjdk9/install.sh
/usr/lib/sdk/openjdk9/install.sh: line 2: mkdir: command not found
/usr/lib/sdk/openjdk9/install.sh: line 4: cp: command not found
/usr/lib/sdk/openjdk9/install.sh: line 5: rm: command not found
/usr/lib/sdk/openjdk9/install.sh: line 6: cp: command not found
Error: module openjdk: Child process exited with code 127

Reply:
Looks like append-path replaces the existing PATH variable, which removes /usr/bin

Reply:
It seems that this lines that I added removed the /usr/bin :
"env": {
      "PATH": "/usr/bin:/app/bin:/usr/lib/sdk/gfortran-62/bin:/usr/lib/sdk/openjdk9/bin"
    }
the "append-path": "/usr/lib/sdk/openjdk9/bin", should do the same, shouldn't? So, I removed "PATH": "/usr/bin:/app/bin:/usr/lib/sdk/gfortran-62/bin:/usr/lib/sdk/openjdk9/bin" and now I get again the "./build.sh: line 3: javac: command not found"

Reply:
The pb should be the the extension is sdk. 

 "org.freedesktop.Sdk.Extension.openjdk9"

Isn't there a jdk extension?

Reply:
Filed an issue: https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk9/issues/3